kik_unofficial/client.py

```diff
@@ -913,10 +913,18 @@ def is_multi_packet(self, data: bytes):
     def cleanup_buffer(self):
         # This method is called after the cleanup_interval to remove outdated data from the buffer
         current_time = time.time()
+        cleaned_count = 0  # Initialize a counter for cleaned messages
+
         self.logger.debug("Cleaning up partial data buffer")
         self.data_buffer = [(ts, data) for (ts, data) in self.data_buffer if current_time - ts <= self.cleanup_interval]
+
+        # Calculate the number of messages cleaned
+        cleaned_count = len(self.data_buffer) - cleaned_count
```

Comment:
Do I miss anything or that `cleaned_count` will always be `len(self.data_buffer`?

Reply:
You are correct, my logic was wrong. I have fixed it so it now sets cleaned_count to len of the array then cleans it. Then subtracts cleaned_count from the length after its been cleaned.